dash/include/dash/halo/StencilOperator.h

```diff
@@ -439,22 +663,22 @@ class StencilOperator {
     return stencil_offs;
   }
 
-  pattern_index_t get_offset(const ElementCoords_t& coords) const {
-    pattern_index_t offset = 0;
-
+  StencilOffsets_t set_dimension_offsets() {
+    StencilOffsets_t      dim_offs;
+    signed_pattern_size_t offset = 0;
```

Comment:
My compiler tells me, that this was declared but never referenced

Reply:
@dhinf Can this function be removed?

Reply:
no, it is used.